Tests/IntegrationTests.Shared/RelationshipTests.cs

```diff
@@ -360,25 +360,43 @@ public void TestAddingStandAloneObjectToToManyRelationship()
             Assert.That(dogAgain, Is.Not.Null);
             Assert.That(dog.IsManaged);
         }
-
-        [Test, Description("This is the case until we support standalone lists")]
+        /*
+        [Test, Description("This is the default C# behaviour for a list property")]
         public void StandaloneListsShouldBeNull()
         {
             var person = new Person();
 
             Assert.That(person.Friends, Is.Null);
         }
-
-        [Test, Ignore("Fails until we support standalone lists")]
+*/
+
+        /*
+         NoteOnListInit
+         If you have a simple braced init like 
+         Friends = { new Person {  FullName = "Sally"} }
+         it requires the property is already initialised and is short-hand for a series of Add
+         calls such as
+         Friends.Add( new Person {  FullName = "Sally"} );
+
+         A null property has to be initialised with a full object:
+         Friends = new List<Person> { new Person {  FullName = "Sally"} }
+
+         We have two approaches under consideration which would allow the simple braced init:
+         1) automatically weave in a clause in the ctor initialising the list properties, 
+            which adds overhead to creating an object if they never use the list.
+         2) If the user chooses to init a list in the ctor, preserve the ctor init (which is a 
+            requested feature anyway in issue 109)
+        */
```

Comment:
On the topic of the issue described in the comment I think a good compromise will be to weave the list getter so that it lazily instantiates the list in the unmanaged case:
```
public IList<Person> Friends
{
    get
    {
        if (!base.IsManaged)
        {
             if (_friends == null)
                 _friends = new List<Person>();

             return _friends;
        }

        return base.GetListValue<Person>(...);
    }
}
```